src/Conductor.Domain/Services/DefinitionService.cs

```diff
@@ -57,5 +57,10 @@ public Definition GetDefinition(string id)
         {
             return _repository.Find(id);
         }
+
+        public IEnumerable<Definition> GetDefinitions()
+        {
+            return _repository.GetAll();
```

Comment:
The pagination should happen in the repo layer and should return an [IAsyncEnumerable](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.iasyncenumerable-1)

Reply:
No worries, I can change this.